Firestore/core/src/firebase/firestore/timestamp_internal.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_TIMESTAMP_INTERNAL_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_TIMESTAMP_INTERNAL_H_
+
+#include "Firestore/core/include/firebase/firestore/timestamp.h"
+
+namespace firebase {
+
+/**
+ * Details about the Timestamp class which are useful internally, but we don't
+ * want to expose publicly. Currently, just a collection of limits.
```

Comment:
Optional nit: I'd delete the _"Currently"_ part, this looks like a comment that is prone to become stale.

Reply:
Done